pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -277,15 +277,16 @@ public static List<ViewFilter> getAllFilters(@NonNull String userId, @NonNull Ma
             if (StringUtils.isNotBlank(parent)) {
                 query = query + SQL_AND_PARENT;
             }
-            query = query.replaceAll("(%#%)", "%" + realm + ",%");
 
             try (PreparedStatement stmt = conn.prepareStatement(query)) {
                 stmt.setString(1, userId);
                 stmt.setString(2, ddpGroupId);
                 stmt.setString(3, ddpGroupId);
                 stmt.setString(4, ddpGroupId);
+                stmt.setString(5, "%," + realm + ",%");
+                stmt.setString(6, "%[" + realm + ",%");
```

Comment:
Did we mean to put `%[` here instead of just `%`?  I'm not familiar with that syntax.